todocore/src/test/java/com/knighttodo/todocore/factories/DayTodoFactory.java

```diff
@@ -20,7 +18,7 @@ public class DayTodoFactory {
     public static final int HARD_SCARY_EXPERIENCE = 37;
     public static final Scariness UPDATED_SCARINESS_TODO = Scariness.SCARY;
     public static final Hardness UPDATED_HARDNESS_TODO = Hardness.HARD;
-
+    public static final String UPDATED_COLOR_TODO = "Green";
```

Comment:
it won't be "Green" there are color codes like "6e6e6e" it's grey color. But I suppose for testing purposes it's fine.